protocols/v2/const-sv2/Cargo.toml

```diff
@@ -10,6 +10,7 @@ repository = "https://github.com/stratum-mining/stratum"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+secp256k1 = { version = "0.28.2", default-features = false, features =["hashes", "alloc","rand","rand-std"] }
```

Comment:
seems that you can remove this dependency

Reply:
What do you think about this?
`
pub const ELLSWIFT_ENCODING_SIZE: usize = secp256k1::constants::ELLSWIFT_ENCODING_SIZE;
`
I thought it would have been a good idea to keep this constant depending from the one in secp256kq (it is what I meant when I added it in the toml, but later I forgot). 
Otherwise, just
`
pub const ELLSWIFT_ENCODING_SIZE: usize = 64;
`
without calling secp256k1 library.